//package com.sincetimes.redwar.game.comnunicate.protocol;

message RceCustomizedActivity
{
	optional string planetId = 1;
	optional string action = 2;//initCA
}